Bottom: d0077a3c6fcf24f404f72500184ce2e0d7277625
Top:    0dccf949dec10fa503b0ea2307bf0c0c6afca8a0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-20 11:04:09 +0100

Refresh of add-a-demonstration-model-for-

---

diff --git a/src/hid/common/hidgl_package_acy_resistor.c b/src/hid/common/hidgl_package_acy_resistor.c
index c863e3d..801f98f 100644
--- a/src/hid/common/hidgl_package_acy_resistor.c
+++ b/src/hid/common/hidgl_package_acy_resistor.c
@@ -222,7 +222,6 @@ resistor_value_to_stripes (ElementType *element, int *st1, int *st2, int *st3, i
   value -= *st1;
 
   value *= 10.;
-  printf ("XXX: Something dies if I don't print this!: %f\n", value);
   *st2 = (int)value;
   value -= *st2;
